core/src/components/cat-input/cat-input.tsx

```diff
@@ -210,11 +215,16 @@ export class CatInput {
           <label htmlFor={this.id} class={{ hidden: this.labelHidden }}>
             <span part="label">
               {(this.hasSlottedLabel && <slot name="label"></slot>) || this.label}
-              {!this.required && (
+              {!this.required && (!this.requiredMarker || this.requiredMarker === 'optional') && (
```

Comment:
remove the first part of the condition and set a default value for the property
@Prop() requiredMarker: 'none' | 'required' | 'optional' = 'optional';

Reply:
same for the other form elements